### PR TITLE
Display a confirmation when removing credentials

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes: #2903.

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7352/

File: jujugui/static/gui/src/app/components/account/credentials/credentials.js [hunk 1]
```diff
@@ -135,7 +136,46 @@ YUI.add('account-credentials', function() {
...
     */
-    _handleDeleteCredential: function(credential) {
+    _handleDeleteCredential: function(credential=null) {
+      this.setState({'removeCredential': credential});
+    },
+
+    /**
+      Handle deleting a credential.
+
+      @method _handleDeleteCredential
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/account/credentials/credentials.js]: Rename to _generateDeleteCredential, or just delete this line.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7354/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7355/

**Comment by huwshimi** [comment]
Thanks @frankban @Lukewh :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**